### DISCUSSION TITLE
how to prevent quartz server start

### DISCUSSION THREAD

**Discussion by SummerZYQ** [quartznet/quartznet#1208] [Q&A]
i am use Quartz.AspNetCore on cluster
i need to stop some server with InStandbyMode, and recorded
when server restart, how to prevent quartz server start or set InStandbyMode before run job.

**Comment by lahma** [comment]
If you use the hosted service package it will auto start for you. If you don't want that behavior just remove the `AddQuartzHostedService` and resolve the the `IScheduler` manually and start/pause when needed.
